### ISSUE TITLE
macOS no SDL_WINDOW flag to allow both window switching and non native resolutions in fullscreen

### ISSUE THREAD

**Issue by SDLBugzilla** [libsdl-org/SDL#3615]
# This bug report was migrated from our old Bugzilla tracker.

**Reported in version:** 2.0.12
**Reported for operating system, platform:** Mac OS X (All), All

# Comments on the original bug report:

On 2020-03-26 17:12:15 +0000,  wrote:

> On OS X using SDL_WINDOW_FULLSCREEN disables window switching using shortcuts such as Command + Tab, or F3.
> 
> SDL_WINDOW_FULLSCREEN_DESKTOP allows use of window switching but does not allow use of non native resolutions like the above option does.
> 
> It would be nice to have one of the flags be able to use both functionalities.

**Comment by slime73** [comment]
I don't believe Apple provides any robust way to do this. Before fullscreen-desktop became popular, a lot of Mac games manually un-fullscreened themselves when certain key combinations were pressed (IIRC). Making SDL do that on its own would probably lead to unintended consequences.
